.travis.yml

```diff
@@ -56,7 +56,7 @@ before_cache:
 
 install:
   - if [[ $TRAVIS_PYTHON_VERSION == 'pypy' ]]; then python -m pip install -U pip; fi
-  - pip install -rtox-requirements.txt
+  - pip --progress-bar ascii install -rtox-requirements.txt
```

Comment:
The option should be after install (it's an option only for install). The build is failing on the previous line, so it's also needed there.

Reply:
And ... this will fail on Python 2.6, at least, with pip 9.0.1.

Here's another solution: celery/celery#4138

```
pip install ... | cat
```